packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -209,6 +209,9 @@ export interface ISummarizerNode {
 	 * that it might have. For example: if datastore "a" contains dds "b", but the
 	 * path is "/a/.channels/b", then the additional path part is ".channels".
 	 * @param snapshot - the base summary to parse
+	 *
+	 * @deprecated - The code now always assumes that all summary nodes have .channels
```

Comment:
Nit: `@deprecated` comments don't want a `-` between the tag and the body. The only TSDoc tags that want that are `@param` blocks.
```suggestion
	 * @deprecated The code now always assumes that all summary nodes have .channels
```